.github/workflows/build.yml

```diff
@@ -121,7 +139,15 @@ jobs:
         restore-keys: ${{ runner.os }}-t-${{ hashFiles('hack/make-rules/tools.mk') }}-go
     - name: Install tools
       run: make install-tools
-    - name: Notebook tests
+    - name: Notebook tests old
```

Comment:
maybe rename: `Notebook tests old k8s version`

Reply:
It is only a step name, doesn't the job name, and I don't want to put specific K8S versions in the action body, only in its header.
Plus, we cannot use env variables in the strategy values. ;-(
Does it make sense ?